packages/fast-components/src/slider-label/slider-label.template.ts

```diff
@@ -0,0 +1,28 @@
+import { html, when, ref } from "@microsoft/fast-element";
+import { SliderLabel } from "./slider-label";
+
+export const SliderLabelTemplate = html<SliderLabel>`
+    <div
```

Comment:
Could this div be the host element instead?

Reply:
Additional thought - this seems like quite a bit of markup for the label. Is it all necessary?Do we need a containier, positioner and a nested span to create a slider label?

Reply:
moved that root div up into host, cleaned up a bit more